src/integrations/third-party/wpml-wpseo-notification.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Yoast\WP\SEO\Integrations\Third_Party;
+
+use Yoast\WP\SEO\Conditionals\Third_Party\WPML_WPSEO_Conditional;
+use Yoast\WP\SEO\Conditionals\WPML_Conditional;
+use Yoast\WP\SEO\Helpers\Short_Link_Helper;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+use Yoast_Notification;
+use Yoast_Notification_Center;
+
+/**
+ * Adds a notification to the dashboard if the WPML plugin is installed,
+ * but the Yoast SEO Multilingual plugin (a glue plugin to make Yoast SEO and WPML work nicely together)
+ * is not.
+ */
+class WPML_WPSEO_Notification implements Integration_Interface {
+
+	/**
+	 * The short link helper.
+	 *
+	 * @var Short_Link_Helper
+	 */
+	protected $short_link_helper;
+
+	/**
+	 * The notification center.
+	 *
+	 * @var Yoast_Notification_Center
+	 */
+	protected $notification_center;
+
+	/**
+	 * The WPML WPSEO conditional.
+	 *
+	 * @var WPML_WPSEO_Conditional
+	 */
+	protected $wpml_wpseo_conditional;
+
+	const NOTIFICATION_ID = 'wpseoml-not-installed';
```

Comment:
The ID is weird, is that intended. And can you mark this as an `@internal` constant. See:
https://github.com/Yoast/developer-docs/blob/a3e2c2336e7de88edde51c206b1453c756ab2ac0/docs/development/standards/constants-usage.md

Reply:
This was a left-over of a previous naming scheme I tried within the added files / classes. I will use the new one instead 🙂.